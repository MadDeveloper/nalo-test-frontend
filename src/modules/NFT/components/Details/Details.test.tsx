import { BrowserRouter } from "react-router-dom"
import renderer from "react-test-renderer"
import { Details } from "."

const nft = {
  id: "1",
  name: "Origins Sculpture",
  totalMinted: 8,
  price: "2.78",
  like: 195,
  forSale: 3,
  timeLeft: "22 hours",
  isHot: true,
  sellerId: 4321,
  image: "/assets/images/nfts/origin-sculpture.jpeg",
}

describe("NFT module > Details", () => {
  it("should match snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Details nft={nft} />
        </BrowserRouter>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
