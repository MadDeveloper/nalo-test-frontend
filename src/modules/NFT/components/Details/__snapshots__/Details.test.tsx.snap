// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NFT module > Details should match snapshot 1`] = `
<div
  className="root"
>
  <div
    className="header"
  >
    <div
      className="nameContainer"
    >
      <h1
        className="name"
      >
        Origins Sculpture
      </h1>
      <span
        className="root hot"
      >
        Hot ðŸ”¥
      </span>
    </div>
    <div
      className="pricing"
    >
      <span
        className="root price"
      >
        2.00
         ETH
      </span>
      <span
        className="root forSale"
      >
        3
         for sale
      </span>
    </div>
  </div>
  <div
    className="footerWrapper"
  >
    <div
      className="footer"
    >
      <span
        className="root secondary minted"
      >
        8
         editions minted
      </span>
      <span
        className="root secondary root big"
      >
        195
         
        <svg
          aria-hidden="true"
          className="root icon"
          fill="none"
          stroke="currentColor"
          strokeWidth={1.5}
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M21 8.25c0-2.485-2.099-4.5-4.688-4.5-1.935 0-3.597 1.126-4.312 2.733-.715-1.607-2.377-2.733-4.313-2.733C5.1 3.75 3 5.765 3 8.25c0 7.22 9 12 9 12s9-4.78 9-12z"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
        </svg>
      </span>
    </div>
  </div>
</div>
`;
